web-react-ts/src/pages/directory/update/LogMutations.ts

```diff
@@ -5,13 +5,11 @@ export const CREATE_HISTORY_SUBSTRUCTURE = gql`
     $churchType: String!
     $servantType: String!
     $churchId: ID!
-    $connectUpwards: Boolean
```

Comment:
Why are you touching this mutation?

Reply:
reason was because it wasn't being initialised in any where it was being called, and it wasnt being used, so it was breaking most of the updates, i think it was something that probably was started initially but not used just yet

Reply:
It's being used and I just edited it yesterday

Reply:
ohh okay will put it back now